### PR TITLE
Add script(s) for issue 1393

### PR DESCRIPTION
**Pull request by dboltovskyi**
ATF Test Scripts to check [#1393](https://github.com/smartdevicelink/sdl_core/issues/1393)

This PR is **[ready]** for review.

### Summary
New ATF scripts to check the issue.

### ATF version
develop

### Changelog
New scripts

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by GetmanetsIrina** [comment]
@theresalech this PR is ready for Livio review

File: test_scripts/Defects/5_1/1393_ListFiles_from_enumerating_through_the_file_system.lua [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        return false, "ListFiles response contains unexpected parameter 'filenames' \n"
+      end
+      return true
+    end)
+  end
+end
+
+local function deleteFile()
+  local cid = common.getMobileSession():SendRPC("DeleteFile", { syncFileName = requestParams.syncFileName })
+  common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
```
Comment (jacobkeeler) [inline test_scripts/Defects/5_1/1393_ListFiles_from_enumerating_through_the_file_system.lua:56]: This doesn't cover the original issue. The issue was that if the OS decided to delete a file in the background, the file would still appear in ListFiles. This should be replaced with an OS call to remove the file.

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

**Comment by VjKlepikov** [comment]
@jacobkeeler Please find an update in [85df65a](https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2605/commits/85df65a6e854e66e7c99d25ab454b6bc0eacb2dc)

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

**Comment by VjKlepikov** [comment]
@jacobkeeler Done in [d6fb14a5](https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2605/commits/d6fb14a51c2438bcfdf84f64cf83716a06ef3c70)

**Review by jacobkeeler — APPROVED** [review]

**Pull request merged**